parlai/crowdsourcing/tasks/turn_annotations_static/util.py

```diff
@@ -24,7 +24,9 @@ def run_static_task(cfg: DictConfig, task_directory: str):
 
     random.seed(42)
 
-    soft_block_qual_name = cfg.mephisto.task.get('task_name', 'turn_annotations_static')
+    soft_block_qual_name = cfg.mephisto.task.get(
```

Comment:
I thought it should be 
```
    soft_block_qual_name = cfg.mephisto.blueprint.get(
        "block_qualification", default_name
    )
```
Since `block_qualification` is already in the blueprint?

Reply:
This is likely the even better option.

Reply:
Yeah, good idea - changing this, and also making this logic and analogous logic in the model chat task more similar